course6/toyapp_thomasdhuys/drawlib/renderer.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <cstdint>
+#include <vector>
+
+#include <displaylib/surface.h>
+#include <displaylib/window.h>
+#include <displaylib/functions.h>
+
+#include "primitives.h"
+#include "render_object.h"
+
+namespace Draw {
+
+class Renderer {
+public:
+	Renderer();
+	void Render(RenderObject ro);
+private:
+	static Display::Window DefaultWindow();
+	static Display::Surface DefaultSurface();
+	Display::Window window;
+	Display::Surface surface;
+};
```

Comment:
I don't see much purpose in this class. It's just a one function, that even doesn't do anything.

Reply:
It's supposed to draw Lines and Triangles from their respective primitives, but I didn't get there yet.